docs/HowItWorks.md

```diff
@@ -47,8 +47,8 @@ foreach ($fileInfos as $fileInfo) {
 
 ### 2.2 Rectify Phase
 
-- When all nodes are ready, applicies iterates all active Rector
-- Each nodes is compared to `$rector->getNodeTypes()` method to see, if this Rector should do some work on it, e.g. is this class name called `OldClassName`?
+- When all nodes are ready, applicies iterates all active Rectors
```

Comment:
Not sure what `applicies` means, or is it a typo?

Reply:
I think it should be `application`

Reply:
Thx, fixed